ui/src/assets/i18n/de.json

```diff
@@ -0,0 +1,1082 @@
+      }
+    },
+    "status": {
+      "title": "Stream Status",
+      "deploying": "Ausrollen",
+      "deployingContent": "Ausrollen wurde initiiert",
+      "deployed": "Ausgerollt",
+      "deployedContent": "Vollständig ausgerollt basierend auf der Anzahl jeder Stream Anwendungs-Eigenschaften",
+      "partial": "Teilweise",
+      "partialContent": "1 oder mehr Anwendungen sind noch nicht ausgerollt.",
+      "incomplete": "Unvollständig",
+      "incompleteContent": "Mindestens 1 oder mehr von jeder Anwendung ist nicht bei der angeforderten Kapazität",
+      "failed": "Fehlgeschlagen",
+      "failedContent": "1 oder mehr Anwendungen haben nicht eine einzelne Instanz ausgerollt",
+      "undeployed": "Zurückgezogen",
+      "undeployedContent": "Absichlich zurückgezogen, oder erstellt aber nicht ausgerollt"
+    },
+    "undeploy": {
+      "title": "Bestätige zurückziehen des Streams",
+      "title2": "Bestätige zurückziehen der Streams",
+      "content": "Diese Aktion wird den Stream <strong>{{ name }}</strong> zurückziehen. Sind Sie sicher?",
+      "content2": "Diese Aktion wird die unten aufgeführten <strong>{{ count }} Stream-Definitionen</strong> zurückziehen. Sind Sie sicher?",
+      "undeploying": "Ziehe gerade Stream(s) zurück",
+      "undeploy": "Zurückziehen des Streams",
+      "undeploy2": "Zurückziehen der Streams",
+      "message": {
+        "successTitle": "Zurückziehen des Streams",
+        "successContent": "Stream \"{{ name }}\" wurden erfolgreich zurückgezogen.",
+        "successTitle2": "urückziehen der Streams",
+        "successContent2": "{{ count }} Stream(s) zurückgezogen.",
+        "errorContent": "Ein Fehler ist beim Zurückziehen von Streams aufgetreten. Bitte prüfen Sie die Server-Logs für mehr Informationen."
+      }
+    },
+    "scale": {
+      "title": "Skaliere Instanze(n) für Anwendungen",
+      "loading": "Hole Stream Laufzeit-Status ab",
+      "description": "Sie können eine Anwendung zwischen 1 und 9 Instanzen skalieren.",
+      "scale": "Skalieren",
+      "scaling": "Skalieren",
+      "name": "Name",
+      "instance": "Instanz",
+      "message": {
+        "loadingErrorContent": "Ein Fehler ist beim holen der Stream Laufzeit-Status aufgetreten. Bitte prüfen Sie die Server-Logs für mehr Informationen.",
+        "successTitle": "Skaliere Stream",
+        "successContent": "{{ name }} Anwendung skaliert zu {{ count }}."
+      }
+    },
+    "create": {
+      "title": "Erstelle Stream(s)",
+      "create": "Erstelle Stream(s)",
+      "modal": {
+        "creating": "Erstelle gerade Stream(s)",
+        "content": "Diese Aktion wird die folgenden <strong>Stream(s)</strong> erstellen:",
+        "create": "Erstelle den/die Stream(s)",
+        "placeholderName": "Stream-Name",
+        "placeholderDescription": "Stream-Beschreibung",
+        "duplicateName": "Doppelter Stream Name, bitte prüfen Sie die Eingabe-Namen.",
```

Comment:
Doppelter Stream Name, bitte prüfen Sie die Eingabe-Namen

Reply:
This is the same 😁